overrides/main.html

```diff
@@ -9,4 +9,7 @@
 <p>
   We released <code>v35</code> of Renovate. Read the <a href="https://docs.renovatebot.com/release-notes-for-major-versions/" target="_blank">Release notes for major versions of Renovate</a> section of the docs to learn what's changed.
 </p>
```

Comment:
```suggestion
```
I think it's okay to show the `v35` announcement for another week or so. But if you want to get rid of it now, apply the suggestion. 😄